src/collection-facets.ts

```diff
@@ -322,146 +325,109 @@ export class CollectionFacets extends LitElement {
         </h1>
         <div class="facet-group-content ${isOpen ? 'open' : ''}">
           ${this.getFacetTemplate(facetGroup)}
+          ${this.searchMoreFacetsLink(facetGroup)}
         </div>
       </div>
     `;
   }
 
   /**
-   * Generate the list template for each bucket in a facet group
+   * Generate the More... link button just below the facets group
+   *
+   * TODO: want to fire analytics?
    */
-  private getFacetTemplate(facetGroup: FacetGroup): TemplateResult {
-    const bucketsNoFavorites = facetGroup.buckets.filter(
-      bucket => bucket.key.startsWith('fav-') === false
+  private searchMoreFacetsLink(
+    facetGroup: FacetGroup
+  ): TemplateResult | typeof nothing {
+    // don't render More... link if the number of facets < this.allowedFacetCount-1
+    if (Object.keys(facetGroup.buckets).length < this.allowedFacetCount - 1)
```

Comment:
what does `this.allowedFacetCount - 1` mean?
can we make a getter with a descriptive name that does this computation? `this.sidePanelFacetCountMax` is definitely not the solution 😄